e2e/addons/addons_test.go

```diff
@@ -213,5 +216,35 @@ var _ = Describe("addons flow", func() {
 				Expect(logLine.IngestionTime).NotTo(Equal(0))
 			}
 		})
+
+		It("svc delete should not delete local files", func() {
+			_, err := cli.SvcDelete(svcName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect("./copilot/hello/addons").Should(BeADirectory())
+			Expect("./copilot/hello/manifest.yml").Should(BeAnExistingFile())
+			Expect("./copilot/.workspace").Should(BeAnExistingFile())
+
+			_, svcInitErr = cli.SvcInit(&client.SvcInitRequest{
```

Comment:
Can you add a comment on why we're re-creating these resources, eventually we might forget why